src/app/pages/Collectibles/components/CollectibleBlur/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+
+import clsx from 'clsx';
+
+import { ReactComponent as RevealEyeSvg } from 'app/icons/reveal-eye.svg';
+
+import BlurImageSrc from './blur.png';
+
+interface Props {
+  onClick?: EmptyFn;
+}
+
+export const CollectibleBlur: FC<Props> = ({ onClick }) => {
+  return (
+    <div
+      onClick={onClick}
+      className={clsx('relative flex justify-center items-center h-full w-full', onClick && 'cursor-pointer')}
+    >
+      <img src={BlurImageSrc} alt="Adult content" className="h-full w-full" />
+      <RevealEyeSvg className="absolute z-10" color="#718096" />
```

Comment:
Please try specifying this color in the `div` above with Tailwind class name and adding here class `stroke-current` or `fill-current`